databricks/koalas/__init__.py

```diff
@@ -137,7 +138,16 @@ def _auto_patch():
         df.DataFrame.to_koalas = DataFrame.to_koalas
 
 
-_auto_patch()
+def _auto_patch_pandas():
+    if sys.version_info >= (3, 7):
+        import pandas as pd
+
+        pd.DataFrame.__class_getitem__ = lambda params: DataFrame.__class_getitem__(params)
+        pd.Series.__class_getitem__ = lambda params: Series.__class_getitem__(params)
```

Comment:
Shall we check if the pandas DataFrame/Series have this method to early detect that pandas supports it in the future.